# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../sysvinit/no_wall.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./src/Makefile.orig	2009-07-15 23:08:02.000000000 +0300
+++ ./src/Makefile	2009-07-15 23:09:35.000000000 +0300
@@ -26,8 +26,8 @@
 ifeq ($(DISTRO),)
 BIN	+= mountpoint
 SBIN	+= sulogin bootlogd
-USRBIN	+= utmpdump wall
-MAN1	+= mountpoint.1 wall.1
+USRBIN	+= utmpdump
+MAN1	+= mountpoint.1
 MAN8	+= sulogin.8 bootlogd.8
 endif
 
